pkg/aws/gw-machineset.go

```diff
@@ -65,12 +65,14 @@ spec:
                 - name: tag:Name
                   values:
                     - {{.InfraID}}-worker-sg
+          {{if not .UseLoadBalancer}}
                     - {{.SecurityGroup}}
           subnet:
             filters:
               - name: tag:Name
                 values:
                   - {{.PublicSubnet}}
+          {{ end }}
```

Comment:
```suggestion
          {{- end }}
```

Here also you need the `-`